http/api/endpoint_test.go

```diff
@@ -83,36 +87,50 @@ func TestPublish(t *testing.T) {
 		"publish message": {
 			chanID:      chanID,
 			msg:         msg,
-			contentType: contentType,
+			contentType: contentTypeSenML,
+			key:         thingKey,
+			status:      http.StatusAccepted,
+		},
+		"publish message with application/senml+cbor content-type": {
+			chanID:      chanID,
+			msg:         msgCBOR,
+			contentType: contentTypeCBOR,
+			key:         thingKey,
+			status:      http.StatusAccepted,
+		},
+		"publish message with application/json content-type": {
+			chanID:      chanID,
+			msg:         msgJSON,
+			contentType: contentTypeJSON,
 			key:         thingKey,
 			status:      http.StatusAccepted,
 		},
 		"publish message with empty key": {
 			chanID:      chanID,
 			msg:         msg,
-			contentType: contentType,
+			contentType: contentTypeSenML,
```

Comment:
Why only SenML, what about the other 3 CTs we support?

Reply:
There are already tests for CTs validation and it's pointless to test 3 CTs for `empty key`.